src/legacy/core_plugins/timelion/public/timelion_vis_fn.ts

```diff
@@ -43,11 +40,11 @@ export const timelionVis = () => ({
     interval: {
       types: ['string', 'null'],
       default: 'auto',
-    }
+    },
   },
-  async fn(context, args) {
+  async fn(context: any, args: any) {
     const $injector = await chrome.dangerouslyGetActiveInjector();
-    const Private = $injector.get('Private');
+    const Private = $injector.get('Private') as any;
```

Comment:
I think this could be either:
```ts
const Private: Function = $injector.get('Private');
```
or possibly
```ts
const Private: IPrivate = $injector.get('Private');
```

Either would be better type safety than casting to `any`

Reply:
The last looks much better. Thanks. Has been added.